### PR TITLE
Added import scale, fixed for 2018.1, and updated readme

### PR DESCRIPTION
**Pull request by BlinksTale**
Now allows for 1x, 2x, 4x, etc scaled assets to be imported with correct positioning. Tweak to now run on 2018.1, with readme updated for how Framer and Framer Generator have changed since March 2018.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,48 @@
...
+4. Proceed to Unity Import section.
 
 ### Import to Unity
 
 * Inside Unity make sure that `Editor Settings/Default Behavior Mode` is set to **2D**, so the assets will be imported as sprites.
 * Open the plugin via `Window/Sketch Import
 * Select the `layers.json` from your *Sketch Export* (inside the `imported` subfolder as mentioned above).
 * Select an UI component (like Canvas,Panel,...) inside your scene as the *Root* for your imported UI layers.
-* Select the Image Prefab from the Import Package Prefabs folder.
+** Note: Selecting Canvas allows you to use "Scale With Screen Size" as the UI scale mode, where the reference resolution can be adjusted so the Sketch file resolution matches the Unity resolution.
```
Comment (fx-lange) [inline README.md]: Formatting is a little off with this additional bullet point and I'm wondering why you removed the point regarding the image prefab. Is this due upgrading to 2018.1 or by accident?

**Review by fx-lange — CHANGES_REQUESTED** [review]
Hi and thanks a lot for your contribution @BlinksTale. 

I can't test your changes as I'm not on Mac anymore but they look reasonable :)

I only have one comment/question regarding the README as you can see below.

File: README.md (hunk 1 above)
Comment (BlinksTale) [reply README.md]: Oh, I can change that bullet formatting back. I'm not sure the image prefab needs? I never saw this in the Sketch Import window options. The only visible options were the layers.json selector, my new Scale option, and the root transform. Selecting  the UI component worked fine for that last one, and then the Import button seemed to work correctly from there.

What did the Image Prefab do previously? Any Canvas seems to work as the UI component now - I haven't tested anything else.

Comment (fx-lange) [reply README.md]: You are right, this point is outdated. The prefab selection is happening automatically.

**Review by fx-lange — APPROVED** [review]

**Pull request merged**